onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -7673,6 +7673,36 @@ def aten_remainder(self: TTensor, other: TTensor) -> TTensor:
     return op.Sub(self, op.Mul(rounded_quotient, other))
 
 
+@torch_op("aten::remainder.Scalar", trace_only=True)
+def aten_remainder(self: TTensor, other: float) -> TTensor:
+    """remainder.Scalar(Tensor self, Scalar other) -> Tensor"""
+
+    other_tensor = ir.tensor(other, dtype=self.dtype)
+
+    if self.dtype.is_integer():
```

Comment:
I think onnx Mod supports float as well but i don't remember why we should not use it here.

Reply:
The rounding behavior is different: https://docs.pytorch.org/docs/stable/generated/torch.remainder.html

Reply:
Created https://github.com/onnx/onnx/issues/7520. Resolving for now